VolumetricConvolution.lua

```diff
@@ -2,95 +2,56 @@ local VolumetricConvolution, parent
    = torch.class('cudnn.VolumetricConvolution', 'nn.VolumetricConvolution')
 local ffi = require 'ffi'
 local errcheck = cudnn.errcheck
+local algo = require 'cudnn.algo'
 
-local autotunerCache = {}
-autotunerCache[1] = {} -- forward
-autotunerCache[2] = {} -- backwardFilter
-autotunerCache[3] = {} -- backwardData
+local Convolution = cudnn.SpatialConvolution
 
+function VolumetricConvolution:__init(nInputPlane, nOutputPlane,
```

Comment:
`cudnn.VolumetricConvolution` does not work for me after this PR was merged, and this looks kind of broken. Where are `dT` and `padT` in the arg list? Where is the call to the parent's __init?

Was VolumetricConvolution code tested before the merge?